src/Calculator/Views/MemoryListItem.xaml

```diff
@@ -59,6 +59,7 @@
                        Style="{ThemeResource TitleTextBlockStyle}"
                        FontWeight="SemiBold"
                        FlowDirection="LeftToRight"
+                       IsTextSelectionEnabled="True"
```

Comment:
As with the history items, recommending `HorizontalAlignment="Right"` here.

Reply:
Good idea!